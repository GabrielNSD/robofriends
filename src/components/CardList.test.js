import React from "react";
import { shallow, mount, render } from "enzyme";
import { create } from "react-test-renderer";
import CardList from "./CardList";

//const testCardList = create(<CardList />);

it("expect to render Card Component", () => {
  const mockRobots = [
    {
      id: 1,
      name: "AAA",
      username: "aaa",
      email: "aaa@mail.com",
    },
  ];
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});

it("expect to render Card Component react test render", () => {
    const mockRobots = [
      {
        id: 1,
        name: "AAA",
        username: "aaa",
        email: "aaa@mail.com",
      },
    ];
    expect(create(<CardList robots={mockRobots} />)).toMatchSnapshot();
  });
  
