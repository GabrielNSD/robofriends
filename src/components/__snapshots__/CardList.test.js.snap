// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card Component 1`] = `
<div>
  <Card
    email="aaa@mail.com"
    id={1}
    key="0"
    name="AAA"
  />
</div>
`;

exports[`expect to render Card Component react test render 1`] = `
<div>
  <div
    className="tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5"
  >
    <img
      alt="robots"
      src="https://robohash.org/1?size=200x200"
    />
    <div>
      <h2>
        AAA
      </h2>
      <p>
        aaa@mail.com
      </p>
    </div>
  </div>
</div>
`;
